From 4324798a00963c6abdc143e540fee2225c597726 Mon Sep 17 00:00:00 2001
From: Suhaas Joshi <s-joshi@ti.com>
Date: Tue, 22 Oct 2024 13:42:35 +0530
Subject: [PATCH] arm64: configs: enable CONFIG_CGROUP_PIDS

The moby check-config script [1] outputs a list of configs that must be
enabled for Docker to work correctly for all Docker images.
CONFIG_CGROUP_PIDS is one such config, which isn't enabled. This is
causing errors in pulling and running certain images. So enable this
config.

[1] https://github.com/moby/moby/blob/master/contrib/check-config.sh

Signed-off-by: Suhaas Joshi <s-joshi@ti.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 7ffb46b74ae5..3286f6c98a35 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -25,6 +25,7 @@ CONFIG_CGROUP_DEVICE=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_PERF=y
 CONFIG_CGROUP_BPF=y
+CONFIG_CGROUP_PIDS=y
 CONFIG_USER_NS=y
 CONFIG_SCHED_AUTOGROUP=y
 CONFIG_BLK_DEV_INITRD=y
-- 
2.40.0

